python/mxnet/symbol/numpy/_symbol.py

```diff
@@ -8012,7 +8012,7 @@ def diagonal(a, offset=0, axis1=0, axis2=1):
 
 # pylint:disable=redefined-outer-name, too-many-arguments
 @set_module('mxnet.symbol.numpy')
-def sum(a, axis=None, dtype=None, out=None, keepdims=None, initial=None, where=None):
+def sum(a, axis=None, dtype=None, out=None, keepdims=False, initial=None, where=None):
```

Comment:
@szha Figured out a bug in `symbol.sum` which does not hybridization. Minimal reproducible:
```python
import mxnet as mx
from mxnet.gluon.nn import HybridBlock
mx.npx.set_np()

class Foo(HybridBlock):
    def hybrid_forward(self, F, x):
        return F.np.sum(x)

foo = Foo()
foo.hybridize()
out = foo(mx.np.ones((10, 10)))
out.asnumpy()
```